src/api/config/clock.rb

```diff
@@ -27,6 +27,10 @@ module Clockwork
     SendEventEmailsJob.perform_later
   end
 
+  every(3.minutes, 'backend_measurements') do
+    BackendMeasurementsJob.perform_later
```

Comment:
3 minutes is how long it takes to expire the backend cached data.

Reply:
It's a bit more complicated I think. There is this tick in `clockwork`

```
every(30.seconds, 'status.refresh') do
  # this should be fast, so don't delay
  WorkerStatus.new.update_workerstatus_cache
end
```

That method *always* fetches the status from the backend, writes it into the cache and creates `StatusHistory`s from the data. That cache is then used throughout the app. So you practically you have a cache that refreshed every 30 seconds and expires (if no one refreshes) in 3 minutes.

Reply:
True, so we can make it 30 seconds or 1 minute if you want. Because we want to monitor it, lower delay among the points would be better.. are you ok with 1 minute here @hennevogel ?

Reply:
I reduce it to `1.minute`

Reply:
I would prefer if we could unify this to 

```
- refresh the cache every 60 seconds
- create StatusHistory every 60 seconds
- send metrics every 60 seconds
```

preferably in one job. Would make all this this way more clear and understandable IMHO.